src/v3/test/integration/__snapshots__/authenticator-piv-cac-verification.test.tsx.snap

```diff
@@ -755,44 +785,14 @@ exports[`authenticator-piv-cac-verification renders form with idpDisplay as SECO
                 class="MuiBox-root emotion-7"
               >
                 <button
-                  class="MuiButtonBase-root MuiButton-root MuiButton-primary MuiButton-primaryPrimary MuiButton-sizeMedium MuiButton-primarySizeMedium MuiButton-disableElevation MuiButton-fullWidth MuiButton-root MuiButton-primary MuiButton-primaryPrimary MuiButton-sizeMedium MuiButton-primarySizeMedium MuiButton-disableElevation MuiButton-fullWidth emotion-31"
+                  class="MuiButtonBase-root MuiButton-root MuiButton-primary MuiButton-primaryPrimary MuiButton-sizeMedium MuiButton-primarySizeMedium MuiButton-disableElevation MuiButton-fullWidth MuiButton-root MuiButton-primary MuiButton-primaryPrimary MuiButton-sizeMedium MuiButton-primarySizeMedium MuiButton-disableElevation MuiButton-fullWidth emotion-37"
                   data-type="save"
                   tabindex="0"
                   type="submit"
                 >
                   Sign in
                 </button>
               </div>
-              <div
-                class="MuiBox-root emotion-7"
-              >
-                <div
-                  class="MuiDivider-root MuiDivider-fullWidth MuiDivider-withChildren separation-line emotion-33"
-                  role="separator"
-                >
-                  <span
-                    class="MuiDivider-wrapper emotion-34"
-                  >
-                    OR
-                  </span>
-                </div>
-              </div>
-              <div
-                class="MuiBox-root emotion-7"
-              >
-                <button
-                  class="MuiButtonBase-root MuiButton-root MuiButton-secondary MuiButton-secondaryPrimary MuiButton-sizeMedium MuiButton-secondarySizeMedium MuiButton-disableElevation MuiButton-fullWidth MuiButton-root MuiButton-secondary MuiButton-secondaryPrimary MuiButton-sizeMedium MuiButton-secondarySizeMedium MuiButton-disableElevation MuiButton-fullWidth  piv-button emotion-36"
-                  tabindex="0"
-                  type="button"
-                >
-                  <span
-                    class="MuiButton-startIcon MuiButton-iconSizeMedium emotion-37"
-                  >
-                    <mocksvgicon />
-                  </span>
-                  Sign in with PIV / CAC card
-                </button>
-              </div>
```

Comment:
Just a reminder here that this may need attention since the test is explicitly testing `idpDisplay` config option feature.